packages/hash/frontend/src/pages/[account-slug]/types/entity-type/[entity-type-id].page.tsx

```diff
@@ -85,31 +87,39 @@ const Page: NextPageWithLayout = () => {
 
   const propertyTypes = useRemotePropertyTypes();
 
+  const getSchemaFromEditorForm = (
+    properties: EntityTypeEditorPropertyData[],
+  ): Partial<EntityType> =>
+    properties.reduce((schema: Partial<EntityType>, property) => {
```

Comment:
Non-blocking: You might want to consider using `for .. of` for this and not a reduce, which might be easier to read

Reply:
Followed this suggestion in 89fd36d88b1c34a1037550fafae2eb287234492b.